### PR TITLE
Feature/feature periodic notification 2

### PR DESCRIPTION
**Pull request by enrichp**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/views/EventView.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePeriodicNotification.java  3
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrNewPeriodicNotification.java  3
- app/src/test/java/org/pesmypetcare/mypetcare/controllers/TestTrNewPeriodicNotification.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/features/pets/PeriodEvent.java  1
- app/src/test/java/org/pesmypetcare/mypetcare/controllers/TestTrDeletePeriodicNotification.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5409665&bid=17736278)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/calendar/CalendarCommunication.java [hunk 1]
```diff
@@ -30,6 +30,10 @@
      */
     void deletePersonalEvent(Pet pet, Event event);
 
+    void newPeriodicNotification(Pet selectedPet, int periodicity, String period, String periodDay);
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/calendar/CalendarCommunication.java:33]: You should comment this methods

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/views/EventView.java [hunk 2]
```diff
@@ -63,7 +64,19 @@ protected String getFirstLineText() {
     }
 
     @Override
-    protected String getSecondLineText() {
-        return DateConversion.getHourMinutes(event.getDateTime());
+    protected String getSecondLineText(Pet pet) {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/views/EventView.java:67]: You should review the override

File: app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePeriodicNotification.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    public void setEvent(Event event) {
+        this.event = event;
+    }
+
+    public boolean isResult() {
+        return result;
+    }
+
+    public void execute() throws ParseException, UserIsNotOwnerException {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePeriodicNotification.java:41]: You should comment  this as well

File: app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrNewPeriodicNotification.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+    public void setPeriodicity(int period) {
+        this.period = period;
+    }
+
+    public boolean isResult() {
+        return result;
+    }
+
+    public void execute() throws ParseException, UserIsNotOwnerException {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrNewPeriodicNotification.java:43]: You should comment this as well

**Review by AlbertPG — COMMENTED** [review]
Take into account these issues

**Pull request closed without merging**